app/src/main/java/io/homeassistant/companion/android/settings/widgets/views/ManageWidgetsView.kt

```diff
@@ -0,0 +1,102 @@
+package io.homeassistant.companion.android.settings.widgets.views
+
+import android.appwidget.AppWidgetManager
+import android.content.Intent
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.material.Text
+import androidx.compose.material.TextButton
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import io.homeassistant.companion.android.common.R
+import io.homeassistant.companion.android.settings.widgets.ManageWidgetsViewModel
+import io.homeassistant.companion.android.widgets.button.ButtonWidgetConfigureActivity
+import io.homeassistant.companion.android.widgets.entity.EntityWidgetConfigureActivity
+import io.homeassistant.companion.android.widgets.media_player_controls.MediaPlayerControlsWidgetConfigureActivity
+import io.homeassistant.companion.android.widgets.template.TemplateWidgetConfigureActivity
+
+@Composable
+fun ManageWidgetsView(
+    viewModel: ManageWidgetsViewModel
+) {
+    LazyColumn(
+        modifier = Modifier.padding(10.dp)
+    ) {
+        if (viewModel.buttonWidgetList.isNullOrEmpty() && viewModel.staticWidgetList.isNullOrEmpty() &&
+            viewModel.mediaWidgetList.isNullOrEmpty() && viewModel.templateWidgetList.isNullOrEmpty()
+        ) {
+            item { Text(stringResource(id = R.string.no_widgets)) }
+        }
+        if (!viewModel.buttonWidgetList.isNullOrEmpty()) {
+            item {
+                Text(stringResource(id = R.string.list_button_widgets))

```

Comment:
I know this hasn't changed, but the "List of" part of the string feels unnecessary and using "Button widgets", "Entity state widgets", etc as headings also work.